### PR TITLE
Update CHANGELOG for the 2019.02.0 release

### PR DESCRIPTION
**Pull request by gonzaiva**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
```
Comment (ryanjulian) [inline CHANGELOG.md]: matplotlib

to TensorBoard

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- Relative Entropy Policy Search (REPS) class
```
Comment (ryanjulian) [inline CHANGELOG.md]: algorithm (not class)

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
+- Vanilla loss for NPO
+- Truncated Natural Policy Gradient (TNPG) class
+- Episodic Reward Weighted Regression (eRWR) class
```
Comment (ryanjulian) [inline CHANGELOG.md]: ERWR

algorithm not class

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
+- Vanilla loss for NPO
+- Truncated Natural Policy Gradient (TNPG) class
```
Comment (ryanjulian) [inline CHANGELOG.md]: algorithm not class

File: CHANGELOG.md [hunk 5]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
+- Vanilla loss for NPO
```
Comment (ryanjulian) [inline CHANGELOG.md]: Vanilla policy gradient loss to NPO

File: CHANGELOG.md [hunk 6]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
 
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
```
Comment (ryanjulian) [inline CHANGELOG.md]: primitives (not class)

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
```
Comment (ryanjulian) [inline CHANGELOG.md]: for TensorFlow-based primitives

File: CHANGELOG.md [hunk 8]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- Update dependencies:
+  - tensorflow-probability to 0.5.x
+  - dm_control to commit 92f9913
+  - TensorFlow to 1.12
+  - MuJoCo to 2.0
+  - gym to 0.10.11
+- Move dm_control tests into the unit test tree
+- Use GitHub standard .gitignore
+- Improve the implementation of RandomizedEnv (Dynamics Randomization)
+- Decouple tensorboard from logger
```
Comment (ryanjulian) [inline CHANGELOG.md]: TensorBoard from the logger

File: CHANGELOG.md [hunk 9]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- Move files from garage/misc/instrument to garage/experiment
+
+### Removed
+- Move some garage subpackages into their own repositories:
+  - garage.viskit to rlworkgroup/viskit
```
Comment (ryanjulian) [inline CHANGELOG.md]:
```markdown
[rlworkgroup/viskit](https://github.com/rlworkgroup/viskit)
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+  - garage.spaces to rlworkgroup/akro
```
Comment (ryanjulian) [inline CHANGELOG.md]:
```markdown
[rlworkgroup/akro](https://github.com/rlworkgroup/akro)
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- Theano branch
```
Comment (ryanjulian) [inline CHANGELOG.md]: It wasn't a branch. This is better phrased as "Remove Theano backend, algorithms, and dependencies"

File: CHANGELOG.md [hunk 12]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- Some environments which duplicated openai/gym
```
Comment (ryanjulian) [inline CHANGELOG.md]:
```markdown
Custom environments which duplicated [openai/gym](https://github.com/openai/gym)
```

File: CHANGELOG.md [hunk 13]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+### Fixed
+- Clean up warnings in the test suite
+- Pickling bug in GaussianMLPolicyWithModel
+- Namescope in LbfgsOptimizer
+- Correctly sample paths in OffPolicyVectorizedSampler
+- Implementation bugs in tf/VPG
+- Bug when importing Box
+- Bug in test_benchmark_her
+
+
```
Comment (ryanjulian) [inline CHANGELOG.md]: should we also add the 2018.10.1 changelog?

File: CHANGELOG.md (hunk 4 above)
Comment (ryanjulian) [reply CHANGELOG.md]: for TensorFlow

File: CHANGELOG.md (hunk 3 above)
Comment (ryanjulian) [reply CHANGELOG.md]: for TensorFlow

File: CHANGELOG.md [hunk 14]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
+- Vanilla loss for NPO
+- Truncated Natural Policy Gradient (TNPG) class
+- Episodic Reward Weighted Regression (eRWR) class
+- gym.Env wrappers used for pixel environments
+- Convolutional Neural Network class
```
Comment (ryanjulian) [inline CHANGELOG.md]: primitive

File: CHANGELOG.md [hunk 15]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
...
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
+  discrete qf-derived policy
+- Base Model class
+- Dump plots generated with Matplotlib in TensorBoard
+- Relative Entropy Policy Search (REPS) class
+- GaussianConvBaseline and GaussianConvRegressor class
+- Dockerfiles for garage
```
Comment (ryanjulian) [inline CHANGELOG.md]: New Dockerfiles, docker-compose files, and Makefiles for running garage using Docker

File: CHANGELOG.md [hunk 16]
```diff
@@ -4,6 +4,55 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2019.02.0
+
+### Added
+- Classes needed by DQN: epsilon greedy strategy, discrete MLP model and
```
Comment (ryanjulian) [inline CHANGELOG.md]: Episolon-greedy exploration strategy, DiscreteMLPModel, and QFunctionDerivedPolicy (all needed by DQN)

File: CHANGELOG.md (hunk 13 above)
Comment (gonzaiva) [reply CHANGELOG.md]: All the back ported changes have been include here, and even one of them doesn't apply anymore (fix mutually exclusive imports of Theano and TensorFlow), so I think it's better to not add it to keep things simple.

Comment (ryanjulian) [reply CHANGELOG.md]: hmm i think we still need to include v2018.10.1 here though. people should be aware that such a release exits. you should be able to just copy/paste

**Pull request merged**